src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier color = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        String randomColor = String.valueOf(color.getRandomColor());
+        int number = new Random().nextInt(100);

```

Comment:
Random should be a class variable
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
        int number = random.nextInt(MAX_NUMBER);
```